src/main/java/com/coveros/selenified/services/HTTP.java

```diff
@@ -194,7 +174,9 @@ public Response post(String service, Request request) {
      * @return Response: the response provided from the http call
      */
     public Response post(String service, Request request, File file) {
-        this.contentType = "multipart/form-data; boundary=" + BOUNDARY;
+        if (file != null) {
+            this.contentType = "multipart/form-data; boundary=" + BOUNDARY;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "multipart/form-data; boundary=" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)